css/style.css

```diff
@@ -0,0 +1,752 @@
+@import url('https://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.cdnfonts.com/css/Lato');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.hidden {
+  display: none;
+}
+
+.hamburger,
+.hamburger-about {
+  color: #333;
```

Comment:
- [ ] To match the original design better, you should consider using the colors provided there. In this case, I think it would look better if you use `#272A31`.
- [ ] Considering that you have a body tag, you should consider adding the color above as your "standard black color".

Reply:
updated